src/main/java/com/JavapracticeApplication.java

```diff
@@ -1,9 +1,7 @@
-package com.javapractice.javapractice;
+package com;
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
-import org.springframework.web.bind.annotation.RestController;
-import org.springframework.web.bind.annotation.GetMapping;
 
 @SpringBootApplication
```

Comment:
This decorator had to be moved to the base part of the app to discover the java packages.